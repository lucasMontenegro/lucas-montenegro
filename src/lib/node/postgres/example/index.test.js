describe(`./index.js`, () => {
  jestUtils.describeDependencies({
    deps: [
      `stringify-object`,
      `bluebird`,
    ],
    relativeBasePath: __dirname,
  })
})