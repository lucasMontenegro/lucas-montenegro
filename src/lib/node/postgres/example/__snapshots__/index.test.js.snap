// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`./index.js external dependencies bluebird should match the installed version 1`] = `"^3.5.5"`;

exports[`./index.js external dependencies stringify-object should match the installed version 1`] = `"^3.3.0"`;
